usaspending_api/awards/v1/views.py

```diff
@@ -182,3 +185,57 @@ def get_queryset(self):
         filtered_queryset = self.filter_records(self.request, queryset=queryset)
         ordered_queryset = self.order_records(self.request, queryset=filtered_queryset)
         return ordered_queryset
+
+
+class FABAFilter(filters.FilterSet):
+
+    class Meta:
+        model = FinancialAccountsByAwards
+        # You can create any relationships you want here using ORM notation
+        fields = ('piid', 'treasury_account__agency_id')
```

Comment:
Can you you showcase surfacing a joined field by an aliased name, rather than having it surface via the dunder syntax?